develop-review/solutions/microservice-speaker/src/main/resources/application.properties

```
@@ -0,0 +1,5 @@
+quarkus.datasource.db-kind = postgresql
+%dev.quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/conference
```

The db name is different from the book
```suggestion
%dev.quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/testing
```

---

oh.. good catch! I renamed the DB when I was creating a generic podman-compose template.